docs/hosting/server-setups/google-cloud.md

```diff
@@ -0,0 +1,146 @@
+# Hosting n8n on Google Cloud
+
+This hosting guide shows you how to self-host n8n on Google Cloud (GCP). It uses n8n with Postgres as a database backend using Kubernetes to manage the necessary resources and reverse proxy.
+
+## Prerequisites
+
+- [The gcloud command line tool](https://cloud.google.com/sdk/gcloud/){:target="_blank" .external-link}
```

Comment:
Does it make sense to set this as a prerequisite if we don't use it in the steps?